app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -164,6 +155,43 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
     }
 
+    private void setupRecyclerView() {
+        RecyclerView recyclerView = binding.rvAttendeeList;
+        recyclerView.setLayoutManager(new LinearLayoutManager(context));
+        recyclerView.addItemDecoration(new DividerItemDecoration(context, DividerItemDecoration.VERTICAL));
+        recyclerView.setItemAnimator(new DefaultItemAnimator());
+
+        FastAdapter<Attendee> fastAdapter = new FastAdapter<>();
+
+        stickyHeaderAdapter = new StickyHeaderAdapter();
+        final HeaderAdapter headerAdapter = new HeaderAdapter();
+        itemAdapter = new ItemAdapter<>();
+
+        fastAdapter.setHasStableIds(true);
+        fastAdapter.withEventHook(new AttendeeItemCheckInEvent(this));
+
+        recyclerView.setAdapter(stickyHeaderAdapter.wrap(itemAdapter.wrap(headerAdapter.wrap(fastAdapter))));
+
+        final StickyRecyclerHeadersDecoration decoration = new StickyRecyclerHeadersDecoration(stickyHeaderAdapter);
+        recyclerView.addItemDecoration(decoration);
+
+        adapterDataObserver = new RecyclerView.AdapterDataObserver() {
+            @Override
+            public void onChanged() {
+                decoration.invalidateHeaders();
+            }
+        };
+        stickyHeaderAdapter.registerAdapterDataObserver(adapterDataObserver);
+    }
+
+    private void setupRefreshListener() {
+        refreshLayout = binding.swipeContainer;
+        refreshLayout.setColorSchemeColors(utilModel.getResourceColor(R.color.colorAccent));
+        refreshLayout.setOnRefreshListener(() -> attendeesPresenter.loadAttendees(true));
```

Comment:
We have defined the progress to show whenever the view is updating. Be it refresh or loading. Hiding progressbar here would not be a great idea as it'll hinder the current workflow. Besides, maybe in future, we'll disable the UI while updating, so a progressbar will be a cue to that. The refresh layout just shows that it's a refresh action